src/main/java/com/example/Customer.java

```diff
@@ -0,0 +1,23 @@
+package com.example;
+
+/**
+ * Public class Customer with the field loyaltyCard and
+ * constructor
+ *
+ * @author Elena Yevsyukova
```

Comment:
Even though there is such a tag, software is a collective enterprise. Tagging code with authorship prevents collective ownership and hinders adding modifications. So, in modern days this tag is deprecated. Please remove.